tests/acceptance/test_acceptance/test_overrides.py

```diff
@@ -36,8 +40,9 @@ def test_overrides(session_obj):
     3. activate experiment and assert forced variation is now in place
     4. Try overriding with the same variation again. Should not be possible.
     """
-    # Confirm deafult variation is "variation_1" (activate)
+    # Confirm default variation is "variation_1" (activate)
     activating = activate_experiment(session_obj)
+    # all overrides tests failed due to deserializing or invalidschema issues
```

Comment:
All overrides tests are also failing due to deseralizing (json.loads) and invalid schema issues